.env.sample

```diff
@@ -15,3 +15,4 @@ VITE_DONATION_ADDRESS=0x6D18a54E0fd87FCb84a0510A3eCd8855b7226715
 VITE_DISCORD_REDIRECT=http://localhost:3000
 
 VITE_CLIENT_VERSION = 2022-04-20T14:58
+VITE_RELEASE_VERSION = local-dev
```

Comment:
@dawsbot are you certain this will accomplish what I'm trying to do? From my research, `process.env.npm_package_version` pulls the version number located in the [package.json](https://github.com/sunflower-land/sunflower-land/blob/main/package.json) file, which is currently set to: `"version": "0.0.0"`. This is not what I want - instead, I want to grab is the latest git commit tag, which at time of writing is `v0.2.18-v1-create-farm-enable`.

Am I mistaken?

Reply:
@dawsbot You make a great point, I think that would be way better. However, this definitely feels like a decision the deployers of the project need to make. Perhaps I create a new [CHORE] issue communicating this, then we close this conversation in this PR?

Reply:
I created a new chore issue here, to be completed by maintainer in the future: https://github.com/sunflower-land/sunflower-land/issues/716